bitrise.yml

```diff
@@ -85,7 +85,7 @@ workflows:
         - project_location: "$BITRISE_FLUTTER_PROJECT_LOCATION/core"
     - flutter-test@1:
         inputs:
-        - additional_params: "--verbose"
+        - additional_params: "--platform chrome --verbose"
```

Comment:
👍

Reply:
why need to add this flag?

Reply:
we use library `dart.html` to download file on browser. But it not found in dart test. So we need add flag to target to web